docs/docs/en/guides/develop/projects/manifests.md

```diff
@@ -63,7 +63,7 @@ Tuist provides <LocalizedLink href="/contributors/principles.html#default-to-con
 ```swift
 import ProjectDescription
 
-let config = Config(
-    generationOptions: .options(enforceExplicitDependencies: true)
+let tuist = Tuist(
```

Comment:
The URL [here](https://github.com/tuist/tuist/pull/7043/files#diff-8f8fef825ad5543b56e538757b9dfadd830c841f623c0ab7c6c75ba507d7b033R61) points to `/references/project-description/structs/config`? I assume that link will no longer exist now that we renamed the model? Should we set up redirects and change the link to `.../structs/tuist`?